### PR TITLE
Add External SNAT parameter

### PR DESCRIPTION
**Pull request by paurosello**

### CODE REVIEW

File: cmd/template/cluster/flag.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ const (
 	flagName                    = "name"
 	flagNoCache                 = "no-cache"
 	flagPodsCIDR                = "pods-cidr"
+	flagExternalSNAT            = "externalsnat"
```
Comment (xh3b4sd) [inline cmd/template/cluster/flag.go:24]:
```suggestion
	flagExternalSNAT            = "external-snat"
```

File: cmd/template/cluster/flag.go [hunk 2]
```diff
@@ -65,6 +67,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.Name, flagName, "", "Tenant cluster name.")
 	cmd.Flags().BoolVar(&f.NoCache, flagNoCache, false, "Force updating release folder.")
 	cmd.Flags().StringVar(&f.PodsCIDR, flagPodsCIDR, "", "CIDR used for the pods.")
+	cmd.Flags().BoolVar(&f.ExternalSNAT, flagExternalSNAT, false, "AWS CNI configuration")
```
Comment (xh3b4sd) [inline cmd/template/cluster/flag.go:70]:
```suggestion
	cmd.Flags().BoolVar(&f.ExternalSNAT, flagExternalSNAT, false, "AWS CNI configuration.")
```

File: cmd/template/cluster/runner.go [hunk 3]
```diff
@@ -71,6 +71,7 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		MasterAZ:          r.flag.MasterAZ,
 		Name:              r.flag.Name,
 		PodsCIDR:          r.flag.PodsCIDR,
+		ExternalSNAT:      r.flag.ExternalSNAT,
```
Comment (xh3b4sd) [inline cmd/template/cluster/runner.go:74]: Can we sort alphabetically please? Experience showed this is easier to work with for humans.

Comment (paurosello) [reply cmd/template/cluster/runner.go]: done

File: cmd/template/cluster/flag.go (hunk 2 above)
Comment (paurosello) [reply cmd/template/cluster/flag.go]: done

File: cmd/template/cluster/flag.go (hunk 1 above)
Comment (paurosello) [reply cmd/template/cluster/flag.go]: changed

**Comment by pipo02mix** [comment]
Thanks for the effort, can you add it in the docs as an example?

**Comment by paurosello** [comment]
Added to the docs

File: docs/template-cluster-cr.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ It supports the following flags:
     specify three distinct availability zones instead. This can be done by separating AZ names with comma or using the flag
     three times with a single AZ name.
   - `--domain`  - base domain of your installation. Customer solution engineer can provide this value.
+  - `--external-snat` - AWS CNI configuration. Only versions *11.3.1+ support this feature.
```
Comment (pipo02mix) [inline docs/template-cluster-cr.md:22]:
```suggestion
  - `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html). Only versions *11.3.1+ support this feature.
```

**Pull request merged**